src/c++11/util.cc

```diff
@@ -11,11 +12,10 @@
 namespace lightstep {
 namespace util {
 
-uint64_t stringToUint64(const std::string& s) {
-  // TODO error handling
+uint64_t hexToUint64(const std::string& s) {
   std::stringstream ss(s);
   uint64_t x;
-  ss >> x;
+  ss >> std::setw(16) >> std::hex >> x;
```

Comment:
For hygiene, since our other libraries format 16-byte, 0-padded guids. I do not believe either option is necessary.